packages/tools/src/tools/FreehandROISculptorTool.ts

```diff
@@ -0,0 +1,685 @@
+import { getEnabledElement } from '@cornerstonejs/core';
+import type { Types } from '@cornerstonejs/core';
+import { BaseTool } from './base';
+import { getAnnotations } from '../stateManagement';
+import {
+  EventTypes,
+  PublicToolProps,
+  ToolProps,
+  SVGDrawingHelper,
+} from '../types';
+import { point } from '../utilities/math';
+import { Events, ToolModes, AnnotationStyleStates } from '../enums';
+import { drawCircle as drawCircleSvg } from '../drawingSvg';
+import { ToolGroupManager } from '../store';
+import { triggerAnnotationRenderForViewportIds } from './../utilities/triggerAnnotationRenderForViewportIds';
+import {
+  hideElementCursor,
+  resetElementCursor,
+} from '../cursors/elementCursor';
+import { StyleSpecifier } from '../types/AnnotationStyle';
+import { getStyleProperty } from '../stateManagement/annotation/config/helpers';
+import { triggerAnnotationModified } from '../stateManagement/annotation/helpers/state';
+
+type CommonData = {
+  selectedIndex: number | null;
+  viewportIdsToRender: any[];
+  isEditingOpenContour: boolean;
+  mouseLocation: Types.Point2 | undefined;
+};
+
+class FreehandROISculptorTool extends BaseTool {
+  static toolName: string;
+  private _commonData?: CommonData = {
+    selectedIndex: null,
+    viewportIdsToRender: [],
+    isEditingOpenContour: false,
+    mouseLocation: undefined,
+  };
+  private _sculptData?: {
+    mousePoint: Types.Point3;
+    mouseCanvasPoint: Types.Point2;
+    points: Array<Types.Point3>;
+    toolSize: number;
+    maxSpacing: number;
+    element: HTMLDivElement;
+  };
+  toolSize: number;
+  isActive = false;
+
+  constructor(
+    toolProps: PublicToolProps = {},
+    defaultToolProps: ToolProps = {
+      supportedInteractionTypes: ['Mouse', 'Touch'],
+      configuration: {
+        maxToolSize: null,
+        minSpacing: 1,
+        referencedToolNames: [
+          'PlanarFreehandROI',
+          'PlanarFreehandContourSegmentationTool',
```

Comment:
Could you switch your tool selection to a pull down in the demo so you actually can test with planar segmentation?

Reply:
Updated example file, now the sculpt tool can be tested with planar segmentation.